tests/test_axis.py

```diff
@@ -168,7 +168,15 @@ class TestCirculationMonitor(DatabaseTest):
     )
 
     def test_process_book(self):
+        config = {
+            Configuration.POLICIES: {
+                Configuration.ANALYTICS_POLICY: ["core.local_analytics_provider"]
+            }
+        }
         with temp_config() as config:
+            Analytics.initialize(
+                ['core.local_analytics_provider'], config
+            )
```

Comment:
I had to change this test because it was relying on local_analytics_provider, and now a previous test is initializing Analytics with a mock_analytics_provider first.